bundler/lib/bundler/man/bundle-config.1.ronn

```diff
@@ -207,6 +207,8 @@ learn more about their operation in [bundle install(1)](bundle-install.1.html).
 * `ignore_messages` (`BUNDLE_IGNORE_MESSAGES`):
    When set, no post install messages will be printed. To silence a single gem,
    use dot notation like `ignore_messages.httparty true`.
+*  `ignore_funding_requests` (`BUNDLE_IGNORE_FUNDING_REQUESTS`):
```

Comment:
Can we move it one entry up to keep it in alphabetical order?

Reply:
done :+1: